src/test/unit/summarize-profile.test.js

```diff
@@ -95,6 +97,7 @@ describe('summarize-profile', function () {
   });
 });
 
-function assertFloatEquals(a, b, message) {
-  assert.ok(Math.abs(a - b) < 0.0001, message || `expected ${a} to be ${b}`);
+function assertFloatEquals(a, b) {
```

Comment:
We should be able to take this function out with `expect(value).toBeCloseTo(0.3);`

https://facebook.github.io/jest/docs/using-matchers.html